types/better-sqlite3/index.d.ts

```diff
@@ -1,89 +1,11 @@
-// Type definitions for better-sqlite3 3.1
+// Type definitions for better-sqlite3 5.0
```

Comment:
If you think the previous version of these type definitions will continue to be updated, you can follow the guidelines found [here](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version).

Reply:
Well, I just modified the version to be related with the current version of better-sqlite3.